.travis.yml

```diff
@@ -13,6 +13,8 @@ env:
   - TOXENV=py2-docs
 #  - TOXENV=py3-docs
   - TOXENV=py2-cover,py3-cover,coverage
+  - TOXENV=py27-pyramid14
+  - TOXENV=py27-pyramid15
```

Comment:
Pyramid 1.6 is unreleased, which means it currently installs pyramid 1.5.7.
I didn't see the point in a second test target running against 1.5.7; once Pyramid 1.6 is out, it's simple to add it.